src/host/os_chdir.c

```diff
@@ -26,6 +24,8 @@ int do_chdir(lua_State* L, const char* path)
 	z = !chdir(path);
 #endif
 
+	(void)(L);  /* warning: unused parameter */
```

Comment:
For some strange reason, only Windows uses it for an error message. I thought it was very strange, I probably should have scoped it to the non-Windows path, I can make that change if you'd like? :)